samples/analyze.v1p3beta1.js

```diff
@@ -0,0 +1,362 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+async function detectPerson(path) {
+  //[START video_detect_person_beta]
```

Comment:
`// [START`

Reply:
Add space

Reply:
Done.

Reply:
... in [this pull request](https://github.com/googleapis/nodejs-video-intelligence/pull/369).